exercises/java-collections/src/main/java/com/cbfacademy/CollectionsExercises.java

```diff
@@ -4,26 +4,80 @@
 
 public class CollectionsExercises {
 
-    public LinkedList<Integer> useLinkedList() {
+
+
+    public static LinkedList<Integer> useLinkedList() {
         // TODO: create an empty linked list
         //  - add 4 as the first element of the list
         //  - then add 5, 6, 8, 2, 9 to the the list
         //  - add another 2 as the last element of the list
         //  - add 4 as the 3rd element of the list
         //  - invoke the method element() on the list and print the result on the screen
         //  - return the list
-        throw new RuntimeException("Not implemented");
+
+        LinkedList<Integer> newList = new LinkedList<>();
+        newList.addFirst(4); // 0
+        newList.add(5);
+        newList.add(6);
+        newList.add(8);
+        newList.add(2);
+        newList.add(9); 
+        newList.addLast(2);
+        newList.add(2, 4);
+
+        int element = newList.element();
+
+        System.out.println(element);
+         
+        return newList;
+
+        
+        //throw new RuntimeException("Not implemented");
     }
 
-    public Stack<Integer> useStack() {
+     public Stack<Integer> useStack() {
+        
         // TODO: create an empty stack
         //  - add 5, 6, 8, 9 to the the stack
         //  - print the first element of the stack on the screen
         //  - print the last element of the stack on the screen
         //  - invoke the method pop() on the stack and print the result on the screen
         //  - invoke the push(4) method on the stack
         //  - return the stack
-        throw new RuntimeException("Not implemented");
+
+        Stack<Integer> stack = new Stack<Integer>();
+
+        stack.add(5);
+        stack.add(6);
+        stack.add(8);
+        stack.add(9);
+        System.out.print(stack.firstElement());
+        System.out.print(stack.lastElement());
+        System.out.print(stack.pop());
+
+
+        /*
+        Aternative- 
+        
+        if(!stack.isEmpty()){
+           int firstElement = stack.firstElement();
+           System.out.print(firstElement);
+
+           int lastElement = stack.lastElement();
+           System.out.print(lastElement);
+
+           int popElement = stack.pop();
+          System.out.print(popElement); 
+        }*/
```

Comment:
Not sure that you need this alternative block. It isn't doing that much that is different from what you already have. 

The if condition is a little redundant since you're adding elements just before then, and therefore wouldn't expect the stack to be empty unless something had gone really wrong.